### ISSUE TITLE
Show detail for games played

### ISSUE THREAD

**Issue by mattmontgomery** [gameshelf/gameshelf.github.io#38]
As a user, I would like to be able to see who I played games with, as well as winners, scores and length.

The following is available from the API:

- Length
- Incomplete
- Should win stats be tracked?
- Location
- Players
  - BGG username & userid
  - Recorded name
  - Starting position
  - Color
  - Score
  - New game for user?
  - Rating
  - Was winner?

**Comment by junqdu** [comment]
🤔 Board Game Stats already does that https://www.bgstatsapp.com/

**Comment by mattmontgomery** [comment]
True! I'm a big fan of the app, and that's where all the data I have for my own plays comes from. It should be a simple implementation to just show that data in some way.
